### PR TITLE
set numpy version to fix deps conflict

### PR DESCRIPTION
**Pull request by VincyZhang**
## Type of Change
fix conda install deps concflict

## Description

detail description 
JIRA ticket: 

## Expected Behavior & Potential Risk

conda install success

## How has this PR been tested?

local test

## Dependency Change?

no

### CODE REVIEW

**Comment by VincyZhang** [comment]
local conda test: https://inteltf-jenk.sh.intel.com/job/lpot-release-wheel-build/16552/

**Comment by VincyZhang** [comment]
local test for 3.9: https://inteltf-jenk.sh.intel.com/job/lpot-release-wheel-build/16563/

File: conda_meta/basic/meta.yaml [hunk 1]
```diff
@@ -14,9 +14,9 @@ requirements:
     - python
     - pip
   run:
-    - python
+    - python<=3.9
```
Comment (chensuyue) [inline conda_meta/basic/meta.yaml:17]: We claim to support python==3.10, so we couldn't limit like this.

**Comment by VincyZhang** [comment]
local test for 3.10: https://inteltf-jenk.sh.intel.com/job/lpot-release-wheel-build/16566/

**Comment by VincyZhang** [comment]
final test：https://inteltf-jenk.sh.intel.com/job/lpot-release-wheel-build/16584/

**Comment by chensuyue** [comment]
Try with [python 3.8](https://inteltf-jenk.sh.intel.com/job/lpot-release-wheel-build/16626/)

**Comment by VincyZhang** [comment]
test with 3.8: https://inteltf-jenk.sh.intel.com/job/lpot-release-wheel-build/16814/

**Review by chensuyue — APPROVED** [review]

**Pull request merged**